systemtests/tests/multi-runscript/after-job.sh

```
@@ -0,0 +1,46 @@
+#   This program is distributed in the hope that it will be useful, but
+#   WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#   Affero General Public License for more details.
+#
+#   You should have received a copy of the GNU Affero General Public License
+#   along with this program; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+#   02110-1301, USA.
+
+#shellcheck source=../../environment.in
+. ./environment
```

Shellcheck (0.11.0)

[info] Not following: ../../environment.in was not specified as input (see shellcheck -x).

(SC1091)